cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/common/TransformDetail.java

```diff
@@ -16,40 +16,40 @@
 
 package co.cask.cdap.etl.common;
 
-import co.cask.cdap.etl.api.StageMetrics;
+import co.cask.cdap.api.metrics.Metrics;
 import co.cask.cdap.etl.api.Transformation;
 
+import java.util.Map;
+
 /**
  * Class that encapsulates {@link co.cask.cdap.etl.api.Transform} and transformId
- * and {@link DefaultStageMetrics}
+ * and {@link Metrics}
  */
 public class TransformDetail {
 
-  private final String transformId;
-  private final Transformation transformation;
-  private final StageMetrics metrics;
+  private final Map<String, Transformation> transformationMap;
+  private final Metrics metrics;
```

Comment:
not sure why we need to pass in metrics anymore if this class is not for a single transform.